crates/ra_syntax/src/ast/generated/nodes.rs

```diff
@@ -1427,7 +2617,10 @@ pub enum ModuleItem {
 impl ast::NameOwner for ModuleItem {}
 impl ast::AttrsOwner for ModuleItem {}
 impl ast::VisibilityOwner for ModuleItem {}
-
+/// Any kind of item that may appear in an impl block
+///
+/// // TODO: is the following a fixme?
+/// impl blocks can also contain MacroCall
```

Comment:
yeah.... I think we should add `MacroCall` to syntax here...

Reply:
Converted this to a fixme in 65b380fa8dfa9efa589f16bfe7e73403c514128e